core/modules/user/components/UserShippingDetails.ts

```diff
@@ -132,21 +125,16 @@ export const UserShippingDetails = {
     fillCompanyAddress () {
       this.useCompanyAddress = !this.useCompanyAddress
```

Comment:
just delete this line, that's the fix 😄 try!
Deleting all this watch, can break something I think, look here: https://github.com/DivanteLtd/vue-storefront/blob/273ab206c0b6733ee10c4fb53188f79289769519/core/modules/user/components/UserShippingDetails.ts#L118

Reply:
Yes, that's the fix :) I wanted to remove `watch` as it is I think less predictable. You are right that this can break something so I've changed it.